examples/index.md

```diff
@@ -290,33 +247,40 @@ Content-Type: application/vnd.api+json
 {
   "errors": [
     {
-      "title": "Invalid Attribute",
       "source": { "pointer": "data/attributes/first-name" },
+      "title": "Invalid Attribute",
       "detail": "First name must contain at least three characters."
     },
     {
-      "title": "Invalid Attribute",
       "source": { "pointer": "data/attributes/first-name" },
+      "title": "Invalid Attribute",
```

Comment:
I gave some thought to the ordering, and decided since I didn't know if there was a preferred order, I wouldn't impose one. That said, I guess it improves readability a bit, so I'll leave this comment here :)